tests/scripts/basic-build-test.sh

```diff
@@ -124,10 +124,10 @@ echo '################ compat.sh ################'
     sh compat.sh -m 'tls1_2 dtls1_2'
```

Comment:
Pre-existing but this is not correct now. It should be `sh compat.sh -m 'tls12 dtls12'` or just `sh compat.sh` as it is supposed to run with defaults. I check the other occurrences of `tls1_2` and it seems that's the only problematic ones.

Reply:
Ah, good catch, will fix. Does this need fixing in 2.28 as well?

Reply:
Yes, it looks like it does. I'll prepare a backport then.

Reply:
2.28 backport: https://github.com/Mbed-TLS/mbedtls/pull/5737 - please review as well.